spec/models/pseud_spec.rb

```diff
@@ -16,4 +16,42 @@
 
   # TODO: add more tests
 
-end
\ No newline at end of file
+  describe "save" do
+
+		before do
+			@user = User.new
+      @user.login = "mynamepseud"
+      @user.age_over_13 = "1"
+      @user.terms_of_service = "1"
+      @user.email = "foo1@archiveofourown.org"
+      @user.password = "password"
```

Comment:
I can see that here, but I've fixed both in the code, and the latest view on Github doesn't show them, so why hasn't Hound's comment disappeared?